configure.ac

```diff
@@ -265,6 +423,7 @@ echo "  ac_configure_args = $ac_configure_args"
 echo "  with tests        = $want_tests"
 echo "  with benchmarks   = $want_benchmark"
 echo "  debug symbols     = $want_debug"
-echo "  with GUI/client   = $build_client"
+echo "  with client       = $build_client"
```

Comment:
I'm a little unsure how I feel about this. It looks like the files have been retooled such that CppBlockUtils is the client, and the GUI is attached to the GUI library. The thing is that the two are basically intertwined. I'm not sure why somebody would want to build one but not the other.

That said, I suppose this is still technically correct in some sense. It might make more sense to name the client something like "Python wrapper," and add a comment pointing out that the Python wrapper and GUI library are required when building the Python frontend.

Reply:
The client by itself would be for armoryd.

Reply:
Good point. I somehow forgot about that. I still think "Python wrapper" is more accurate but I won't hold anything up over it.